e2e/specs/components_html.spec.ts

```diff
@@ -0,0 +1,46 @@
+/**
+ * @license
+ * Copyright 2018-2020 Streamlit Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/// <reference types="cypress" />
+
+describe("components.html", () => {
+  before(() => {
+    // Increasing timeout since we're requesting an external audio file
+    Cypress.config("defaultCommandTimeout", 10000);
```

Comment:
looks like a copy paste that can be cleaned up?

Reply:
That shouldn't be there at all :D